### DISCUSSION TITLE
[Idea] launch glpi-agent monitor right after the glpi-agent installation

### DISCUSSION THREAD

**Discussion by kintaro1981** [glpi-project/glpi-agent#491] [Ideas]
### Your idea

I noticed that once Glpi-Agent is installed with AGENTMONITOR=1 the GLPI Agent Monitor is installed but not launched.
It starts only after a reboot (or maybe a logoff?).

How about launching it right after a successfully installation?

**Comment by g-bougard** [comment]
Hello @kintaro1981 

it is indeed started after a logoff/logon.

Imho, as the installation can be silent (and it is often silent), we can't always decide if we should still start it.
